### PR TITLE
Add split panes to mentoring session

### PR DESCRIPTION
**Pull request by kntsoriano**
https://github.com/exercism/v3-beta/issues/80

@iHiD Could you check if I did the css correctly? Also, I've added `leftMinWidth` and `rightMinWIdth` as props to `<SplitPane />`. You might want to tweak that, too.

### CODE REVIEW

**Comment by kntsoriano** [comment]
Screen Shot 2021-07-12 at 8 04 07 AM

I just noticed this. The markdown editor overflows because it has min-width: 520px

File: app/javascript/components/common/SplitPane.tsx [hunk 1]
```diff
@@ -1,15 +1,32 @@
 import React, { useState, useRef, useEffect, useCallback } from 'react'
+import { useLocalStorage } from '../../utils/use-storage'
 
 export const SplitPane = ({
+  id,
   left,
   right,
+  className = '',
+  leftMinWidth = 100,
+  rightMinWidth = 100,
```
Comment (iHiD) [inline app/javascript/components/common/SplitPane.tsx:10]: Set this to be 620

Comment (iHiD) [reply app/javascript/components/common/SplitPane.tsx:10]: In the place where it's called (not here)

File: app/javascript/components/mentoring/Session.tsx [hunk 2]
```diff
@@ -112,98 +113,105 @@ export const Session = (props: SessionProps): JSX.Element => {
...
-                  status={status}
-                  discussion={discussion}
-                  iterations={iterations}
-                  student={student}
-                  userHandle={userHandle}
-                  onIterationScroll={handleIterationScroll}
+    <SplitPane
+      id="mentoring-session"
+      className="c-mentor-discussion"
+      defaultLeftWidth="50%"
```
Comment (iHiD) [inline app/javascript/components/mentoring/Session.tsx]: minwidth=620 here

File: app/javascript/components/student/MentoringSession.tsx [hunk 3]
```diff
@@ -88,53 +89,60 @@ export const MentoringSession = ({
...
-      <div className="lhs">
-        <header className="discussion-header">
-          <CloseButton url={links.exercise} />
-          <SessionInfo track={track} exercise={exercise} mentor={mentor} />
-          {discussion ? (
-            <DiscussionActions
+    <SplitPane
+      id="mentoring-session"
+      className="c-mentor-discussion"
+      defaultLeftWidth="50%"
```
Comment (iHiD) [inline app/javascript/components/student/MentoringSession.tsx]: And minrightwidth=620 here

**Pull request merged**